### PR TITLE
Add analyzer comments for Elixir lasagna

### PR DESCRIPTION
**Pull request by angelikatyborska**
I'm not 100% if this is the best phrasing for this suggestion.

### CODE REVIEW

File: analyzer-comments/elixir/lasagna/function_reuse.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# function reuse
+
+Proper function reuse is crucial to writing maintainable code.
```
Comment (neenjaw) [inline analyzer-comments/elixir/lasagna/function_reuse.md:3]:
```suggestion
Reusing a written function can help when writing maintainable code.
```

File: analyzer-comments/elixir/lasagna/function_reuse.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+
+Reuse existing functions:
+- `remaining_minutes_in_oven/1` should call `expected_minutes_in_oven/0`
```
Comment (neenjaw) [inline analyzer-comments/elixir/lasagna/function_reuse.md:6]:
```suggestion
- `remaining_minutes_in_oven/1` can reuse `expected_minutes_in_oven/0`
```

File: analyzer-comments/elixir/lasagna/function_reuse.md [hunk 3]
```diff
@@ -0,0 +1,7 @@
...
+- `total_time_in_minutes/2` should call `preparation_time_in_minutes/1`
```
Comment (neenjaw) [inline analyzer-comments/elixir/lasagna/function_reuse.md:7]:
```suggestion
- `total_time_in_minutes/2` can reuse `preparation_time_in_minutes/1`
```

**Review by neenjaw — COMMENTED** [review]
Wording suggestion

File: analyzer-comments/elixir/lasagna/function_reuse.md [hunk 4]
```diff
@@ -0,0 +1,7 @@
+# function reuse
+
+Reusing a written function can help when writing maintainable code.
```
Comment (angelikatyborska) [inline analyzer-comments/elixir/lasagna/function_reuse.md:3]:
```suggestion
Reusing an already written function can help when writing maintainable code.
```

**Review by neenjaw — APPROVED** [review]

**Pull request merged**